### PR TITLE
Rtt remove features

### PR DESCRIPTION
**Pull request by mbouron**

### CODE REVIEW

File: libnopegl/src/node_colorstats.c [hunk 1]
```diff
@@ -149,6 +149,8 @@ static int setup_waveform_compute(struct colorstats_priv *s, const struct pgcraf
                                   const struct ngl_node *texture_node)
 {
     struct texture_priv *texture_priv = texture_node->priv_data;
+    texture_priv->params.usage |= NGLI_TEXTURE_USAGE_SAMPLED_BIT;
```
Comment (ubitux) [inline libnopegl/src/node_colorstats.c:152]: Can you explain in the commit description why textures shouldn't be sampled by default?

File: libnopegl/src/node_graphicconfig.c [hunk 2]
```diff
@@ -242,11 +242,11 @@ static int graphicconfig_init(struct ngl_node *node)
 
 #define COPY_PARAM(name) do {        \
     if (o->name != -1) {             \
-        pending->name = o->name;     \
+        state->name = o->name;     \
```
Comment (ubitux) [inline libnopegl/src/node_graphicconfig.c:222]: style

File: libnopegl/src/node_rtt.c [hunk 3]
```diff
@@ -201,6 +189,14 @@ static int get_renderpass_info(const struct ngl_node *node, int state, struct re
             if (state == RENDER_PASS_STATE_STOPPED)
                 info->nb_interruptions++;
             state = RENDER_PASS_STATE_STARTED;
+        } else if (child->cls->id == NGL_NODE_GRAPHICCONFIG) {
+            struct graphics_state graphics_state = {0};
+            ngli_node_graphicconfig_get_state(child, &graphics_state);
+            if (graphics_state.depth_test)
+                info->require_depth_buffer = 1;
+            if (graphics_state.stencil_test)
+                info->require_stencil_buffer = 1;
```
Comment (ubitux) [inline libnopegl/src/node_rtt.c:198]: I'd prefer to keep a features mask for requirements instead of bools

File: libnopegl/nodes.specs [hunk 4]
```diff
@@ -648,16 +648,6 @@
...
-    "framebuffer_features": [
-      {
-        "name": "depth",
-        "desc": "add depth buffer"
-      },
-      {
-        "name": "stencil",
-        "desc": "add stencil buffer"
-      }
-    ],
```
Comment (ubitux) [inline libnopegl/nodes.specs:660]: probably need a mention in the changelog

File: libnopegl/src/node_graphicconfig.c (hunk 2 above)
Comment (mbouron) [reply libnopegl/src/node_graphicconfig.c]: Fixed.

File: libnopegl/nodes.specs (hunk 4 above)
Comment (mbouron) [reply libnopegl/nodes.specs:660]: Done.

File: tests/meson.build [hunk 5]
```diff
@@ -276,8 +276,6 @@ foreach backend : backends
   tests_rtt = [
     'load_attachment',
     'load_attachment_nested',
-    'feature_depth',
-    'feature_depth_stencil',
```
[the remark this answers is not in the document]
Comment (mbouron) [reply tests/meson.build]: This was a leftover from a previous iteration. Removed.

File: libnopegl/src/node_colorstats.c (hunk 1 above)
Comment (mbouron) [reply libnopegl/src/node_colorstats.c]: Commit dropped. It will have its dedicated PR.

File: libnopegl/src/node_rtt.c (hunk 3 above)
Comment (mbouron) [reply libnopegl/src/node_rtt.c]: Done.

**Review by ubitux — APPROVED** [review]

**Pull request merged**